### ISSUE TITLE
Editor placeholder multiline text - use ellipsis instead

### ISSUE THREAD

**Issue by roblourens** [microsoft/vscode#211403]
First, I'm not sure I like the placeholder pushing the chat input to multiple lines. And this differs from the decoration that I use in the case where there is other text in the input (https://github.com/microsoft/vscode/issues/208976#issuecomment-2078505023)

Also, the editor height doesn't react quite properly to text changes, see the recording

![Recording 2024-04-25 at 19 17 21](https://github.com/microsoft/vscode/assets/323878/81cb81fa-342c-454e-9a22-eb0e090b4411)

**Comment by roblourens** [comment]
Tacking onto this, there is already a placeholder color defined, `input.placeholderForeground`, but this is using the ghost text color.

**Comment by roblourens** [comment]
@daviddossett agreed that the chat input shouldn't grow, but should overflow with `...`. That is something that is hard to implement with a decoration, but might be hard for you to implement too. But it would look nicer than what we do now, which is the text just running into the submit button and getting cut off.

**Comment by hediet** [comment]
Verification steps:
* [Open Playground example.](https://microsoft.github.io/monaco-editor/playground.html?source=v0.50.0-dev-20240607#XQAAAAIXAgAAAAAAAABBqQkHQ5NjdNxly-jY7SIQ9S7DNlzs5W-mwj0fe1ZCDRFc9ws9XQE0SJE1jc2VKxhpR2PdVJZaNO2pgpVvaBoWRH6T5Qp5mb3MVGHlh5tjDZFHy7w2GzLvCkSWyox-NeBn7lum1fgEVTxdzcBJo1gS-gQ0X06zxcafyuQ1_PF0C_bgVsQ1m3AsDp3Qf-qrPC1z88UQv-T6RqsmKPrxYMMnzf-K-d1njFN_zfwJRw8P-LqSGV44c9CxMa2DetDg0XqAEq87g2yD7HTbzTLHKy_M9VPzyq6D4_Vlf5T2qS1sJD7GIIuoAuTVm8K0qeD16yuBxpyJMc1X5r0dHr-dnIUTuuX-sa4XId_HIYXBQqp6YGCGBuTLcjJ1kj8fptS1YDt2iMmHq_5fN2h4KeaEDrdn1MMW_-8dRxmmlitleqsXOt3aI_g4Rq5D5hrtvLc7ZyUn_SyEFjvo8XHCdxymiWuSyfmyIGTunodqIkjw-ODrtWKeLwbLbNiXNaPuQzTFpD__3UZOhQ)
* Notice the scrollbar and missing ellipsis
* Chose the newest nightly monaco editor version
* Verify that there is an ellipsis

**Comment by lramos15** [comment]
![Image](https://github.com/user-attachments/assets/538c99cf-f730-4335-b1f8-ae3ad05e000d)

Not sure if I'm doing something wrong but I don't see ellipses even on latest monaco playground. Will reopen for now since steps are already provided
